pkg/services/navtree/navtreeimpl/navtree.go

```diff
@@ -202,6 +202,18 @@ func (s *ServiceImpl) getHomeNode(c *contextmodel.ReqContext, prefs *pref.Prefer
 		Icon:       "home-alt",
 		SortWeight: navtree.WeightHome,
 	}
+	ctx := c.Req.Context()
+	if s.features.IsEnabled(ctx, featuremgmt.FlagHomeSetupGuide) {
+		var children []*navtree.NavLink
+		// setup guide
+		children = append(children, &navtree.NavLink{
+			Id:         "home-setup-guide",
+			Text:       "Setup guide",
+			Url:        homeUrl + "/setup-guide",
```

Comment:
Looks like in the designs https://www.figma.com/design/q07Y31ld743hnUf2W0TVxv/Growth---New-Setup-Guide?node-id=143-4801&m=dev

@bruno-caldeira1 Bruno wanted a rocket Icon in the entry point. Take a look at the example on this file at line `289` where `Icon` is used and see if you can get that to work? Not sure where the icons are pulled from here though...

Reply:
@CatPerry yea, submenu items don't get icons. Already checked in this [thread](https://raintank-corp.slack.com/archives/C01LENC4MD5/p1725897142285999?thread_ts=1725897002.106089&cid=C01LENC4MD5)